src/material/button/_button-theme.scss

```diff
@@ -151,17 +116,17 @@
     }
   }
 
-  .mat-mdc-unelevated-button {
-    $surface: inspection.get-theme-color($theme, background, card);
-    $primary: inspection.get-theme-color($theme, primary);
-    $accent: inspection.get-theme-color($theme, accent);
-    $error: inspection.get-theme-color($theme, warn);
+  $surface: inspection.get-theme-color($theme, background, card);
+  $primary: inspection.get-theme-color($theme, primary);
+  $accent: inspection.get-theme-color($theme, accent);
+  $error: inspection.get-theme-color($theme, warn);
 
-    $on-surface: on-color($theme, $surface);
-    $on-primary: on-color($theme, $primary);
-    $on-accent: on-color($theme, $accent);
-    $on-error: on-color($theme, $error);
+  $on-surface: on-color($theme, $surface);
```

Comment:
It's not something that was introduced in this PR, but I still this we should fix it: `on-color` should be renamed to `_on-color` so it doesn't get exposed to as a public API.